_includes/releases/v22.2/v22.2.0-beta.1.md

```diff
@@ -0,0 +1,83 @@
+
+<h3 id="v22-2-0-beta-1-db-console-changes">DB Console changes</h3>
+
+- On the SQL Activity Session Details page the Most Recent Statement show the last active query instead of No Active Statement. [#88057][#88057]
+- Update filter labels from "App" to "Application Name" and from "Username" to "User Name" on SQL Activity pages. [#88144][#88144]
+- The **Transaction Execution ID** column now exists in the blocked statements table in the active stmt details page [#88135][#88135]
+
+<h3 id="v22-2-0-beta-1-bug-fixes">Bug fixes</h3>
+
+- This patch fixes incorrect results from queries which utilize locality-optimized search on the inverted index of a table with `REGIONAL BY ROW` partitioning. [#88114][#88114]
+- The function `pg_catalog.pg_get_viewdef` now works properly for materialized views. [#88143][#88143]
+- Cluster and tenant restores of dropped or adding tenants would incorrectly activate those tenants during restore. [#88157][#88157]
+- The `current_setting` builtin function now properly does not result in an error when checking a custom session setting that does not exist and the `missing_ok` argument is true. [#88158][#88158]
+- `CREATE TABLE foo as (SELECT * FROM crdb_internal.check_consistency(...))` no longer panics the gateway node. [#88227][#88227]
+- Previously, CockroachDB could incorrectly not fetch rows with `NULL` values when reading from the unique secondary index when multiple column families are defined for the table and the index doesn't store some of the `NOT NULL` columns. [#88210][#88210]
+- Fixed crashes in changefeed code when running on recent Go versions. [#88231][#88231]
+- For statement execution insights, the value for the end time was updated to be congruent with the start time and latency. [#88276][#88276]
+- Completed stmt diagnostics bundles now persist in the UI in stmt diag bundle pages [#88286][#88286]
+- Dropping temporary tables and sequences should check privilege too. [#88305][#88305]
+- The pgwire `DESCRIBE` step no longer fails with an error while attempting to look up cursors declared with names containing special characters. [#88316][#88316]
+- Jobs pages refreshes data at an interval of 10s [#88285][#88285]
+- Fixed a crash in changefeed expressions when JSON encoding an empty projection. [#88311][#88311]
+- CockroachDB now more promptly reacts to query cancellations (e.g. due to statement timeout being exceeded) after the query spilled to disk. [#88378][#88378]
+
+
+{% comment %}Docs team: Please add these manually.{% endcomment %}
+
+<h3 id="v22-2-0-beta-1-contributors">Contributors</h3>
+
+This release includes 54 merged PRs by 30 authors.
+We would like to thank the following contributors from the CockroachDB community:
+
+- Mark Sirek (first-time contributor, CockroachDB team member)
```

Comment:
Check the [authors file](https://github.com/cockroachdb/cockroach/blob/master/AUTHORS) and remove any CRL employees from this list. If you remove them all, also remove line 56.